tls/s2n_record_read.c

```diff
@@ -221,52 +221,52 @@ int s2n_record_parse(struct s2n_connection *conn)
 
     /* Decrypt stuff! */
     switch (cipher_suite->record_alg->cipher->type) {
-    case S2N_STREAM:
-        GUARD(cipher_suite->record_alg->cipher->io.stream.decrypt(session_key, &en, &en));
-        break;
-    case S2N_CBC:
-        /* Check that we have some data to decrypt */
-        ne_check(en.size, 0);
-
-        /* ... and that we have a multiple of the block size */
-        eq_check(en.size % iv.size, 0);
-
-        /* Copy the last encrypted block to be the next IV */
-        if (conn->actual_protocol_version < S2N_TLS11) {
+        case S2N_STREAM:
+            GUARD(cipher_suite->record_alg->cipher->io.stream.decrypt(session_key, &en, &en));
+            break;
+        case S2N_CBC:
```

Comment:
Was this style change intentional? Just curious because I've noticed that other switch statements in the code base are not indented.